docs/tempo/website/release-notes/v1-5.md

```diff
@@ -0,0 +1,163 @@
+---
+title: V1.5
+weight: 70
+---
+
+# Version 1.5 release notes
+
+The Tempo team is pleased to announce the release of Tempo 1.5.
+
+With this release, Tempo adds Parquet block support (experimental), anonymous usage reporting to help the Tempo team and Grafana understand how Tempo is used, and enhancements to the metrics-generator. 
+
+In addition to the items detailed below, the [application performance management (APM) dashboard](https://grafana.com/docs/tempo/latest/metrics-generator/app-performance-mgmt/) visualizes span metrics (traces data for rates, error rates, and durations (RED)) and service graphs in Grafana 9.1.
+
+## Features and enhancements
+
+Version 1.5 contains the following features and enhancements. 
+
+### Metrics-generator enhancements
+
+The [metrics-generator]({{< relref "../metrics-generator/" >}}) now includes databases and requests across messaging systems in service graph metrics. 
+The OpenTelemetry semantic conventions are used to detect when a span is involved in one of these systems. [PR 1576](https://github.com/grafana/tempo/pull/1576)
+
+### Apache Parquet block format support
+
+Tempo now has a columnar block format based on Apache Parquet.
+A columnar block format has many advantages within Tempo such as enabling faster search but also downstream by enabling a large ecosystem of tools access to the underlying trace data.
+
+Parquet fits all of the requirements:
+
+* Roundtrippable - Support for hierarchical data means regenerating OTLP is straightformat
+
+* More efficient - only read the columns needed for the search
+
+* Faster - only read the columns needed
+
+* Similar or better block size - About a ~5% smaller block size for parquet
+
+* Object storage - Parquet is a file-based approach which translates well to Tempo's block-based design
+
+* Shardable - Parquet naturally includes sub-block structures such as row groups and column chunks which can be processed independently.
+
+This feature is experimental.
```

Comment:
Do we have a pointer on where to turn this on if folks want to experiment?

Reply:
Added information to the release notes on Parquet. I'm also going to write a summary of Parquet for the docs so we'll have something there.